src/components/smart/OeActorWidgetGrid.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <div class="vl-u-flex vl-u-flex-direction-column">
+    <div class="vl-grid">
+      <div class="vl-col--1-1 vl-u-flex vl-u-flex-align-flex-end">
+        <vl-search
+          id="menu-search"
+          name="menu-search"
+          mod-inline
+          mod-alt
+          placeholder="Geef een zoekterm in"
+          @click="handleSearchClick"
+          @submit="triggerSearch"
+          @input="zoekterm = $event.target.value"
+        />
+      </div>
+    </div>
+    <div class="vl-grid vl-u-spacer-top--xsmall">
+      <div class="vl-col--1-1">
+        <span class="vl-u-mark--info vl-u-text--small">{{ rowCountText }}</span>
+        <div class="menu-controls vl-u-float-right">
+          <a :href="`${service?.API_URL}/beheer#/actoren/aanmaken`" target="_blank">
+            <vl-button icon="plus" mod-icon-before mod-naked> Actor aanmaken </vl-button>
+          </a>
+          <vl-button icon="synchronize" mod-icon-before mod-naked @click="search">Vernieuwen</vl-button>
+        </div>
+      </div>
+    </div>
+    <div class="vl-grid vl-u-spacer-top--xsmall">
+      <div class="vl-col--1-1">
+        <oe-grid
+          class="table"
+          :grid-options="gridOptions"
+          @grid-ready="onGridReady"
+          @first-data-rendered="firstDataRendered"
+          @row-clicked="gridOptions?.onRowClicked"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import OeGrid from '../dumb/OeGrid.vue';
+import OeActorWidgetGridActies from './OeActorWidgetGridActies.vue';
+import { VlButton, VlSearch } from '@govflanders/vl-ui-design-system-vue3';
+import { isEmpty, omitBy } from 'lodash';
+import { computed, getCurrentInstance, ref } from 'vue';
+import type { ColDef, FirstDataRenderedEvent, GridOptions, IGetRowsParams, RowClickedEvent } from 'ag-grid-community';
+import type { IActor } from '@models/actor';
+import type { ActorService, IActorenQuery } from '@services/actor.service';
+
+interface IOeActorWidgetGridProps {
```

Comment:
Kunt ge die interfaces apart zetten ergens?

Reply:
Waarom zou die apart moeten staan als die enkel hier gebruikt wordt?

Reply:
Puur preference en aangezien we dat nooit echt doen in onze andere projecten